site/blog/2023/v2.11.md

```diff
@@ -0,0 +1,23 @@
+---
+title: v2.11
+date: 2023-01-13
+tags:
+    - releases
+---
+
+## [v2.11](https://github.com/saulpw/visidata/releases/tag/v2.11) (2023-01-13)
+
+Hi, friends of VisiData!
+
+I’m happy to announce a new maintenance release 2.11. This will be the last release before some big, exciting changes introduced in 3.0.
+
+The most significant change is that 2.11 *drops support for Python 3.6* and *adds support for Python 3.11*.
+
+2.11 also includes several other small improvements and bug fixes, of which I’ll highlight two:
+
+- `dup-sheet` now carries over attributes of columns added by `add-column` (helpful!)
+ - We added a new "concat" jointype. This behaves similar to "append", but keeps first sheet type and columns.
+
```

Comment:
Should we add that unless something comes up in the next 2 weeks or so, this is the version that we want to upload to Debian/Homebrew?

Reply:
Not my call, but that sounds like a great idea to me. Very good to push 2.11 (as the last release before the big 3.0) to Debian/Homebrew and make updating easier for those users.